vscode-antimony/src/server/stibium/stibium/analysis.py

```diff
@@ -430,6 +434,59 @@ def handle_annotation(self, scope: AbstractScope, annotation: Annotation):
         self.table.insert(qname, SymbolType.Parameter)
         self.table.insert_annotation(qname, annotation)
     
+    def get_annotation_descriptions(self):
+        for scope, annotation in self.pending_annotations:
+            self.get_annotation_description(scope, annotation)
+    
+    def get_annotation_description(self, scope: AbstractScope, annotation: Annotation):
+        name = annotation.get_var_name().get_name()
+        qname = QName(scope, name)
+        symbol = self.table.get(qname)
+        if symbol:
+            uri = annotation.get_uri()
+            if uri[0:4] != 'http':
+                return
+            if uri in symbol[0].queried_annotations.keys():
+                return
+            uri_split = uri.split('/')
+            website = uri_split[2]
+            if website == 'identifiers.org':
+                if uri_split[3] == 'chebi':
+                    chebi = ChEBI()
+                    res = chebi.getCompleteEntity(uri_split[4])
+                    name = res.chebiAsciiName
+                    definition = res.definition
+                    queried = '\n{}\n\n{}\n'.format(name, definition)
+                    symbol[0].queried_annotations[uri] = queried
+                else:
+                    return
+                    # uniport = UniProt()
+            elif website == 'www.rhea-db.org':
+                rhea = Rhea()
+                df_res = rhea.query(uri_split[4], columns="equation", limit=10)
+                equation = df_res['Equation']

```

Comment:
Columns in dataframes should be defined at the top of the module as global constants.